src/Carbon/Carbon.php

```diff
@@ -1650,6 +1650,66 @@ public function isTomorrow()
     }
 
     /**
+     * Determines if the instance is within the next week
```

Comment:
All methods descriptions should finish with a point:
`Determines if the instance is within the next week.` ;-)